go.mod

```diff
@@ -10,6 +10,9 @@ require (
 	cosmossdk.io/errors v1.0.0
 	cosmossdk.io/log v1.2.1
 	cosmossdk.io/math v1.2.0
+	cosmossdk.io/server/v2/appmanager v0.0.0-00010101000000-000000000000
```

Comment:
runtime should probably become its own go.mod at this point. Should we call it runtime/v2?

Reply:
Works for the go.mod. I thought we may want to move the current runtime directly to server/v2. Trying to have runtime minimally breaking for app_v2.
Otherwise, we'll endup with app v1 / app v2 for baseapp and sort of the same for server/v2 (?)

Reply:
we can do that, then alias. appv2 isnt really appv2 but app_depinject when you think about it. all the functionality is the same. I think we made a mistake calling it v2. We should rename it through out then go with app v2 as the one with server/v2. That way we can move runtime into server, but something needs to stay and/or be aliased in order not to break modules

Reply:
I'd argue that it is slightly different as one uses runtime and the other not. But the reasoning makes sense. Will update to reflect that.